_includes/releases/v22.1/v22.1.14.md

```diff
@@ -0,0 +1,62 @@
+## v22.1.14
+
+Release Date: February 6, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-14-sql-language-changes">SQL language changes</h3>
+
+- [`COPY`](../v22.1/copy-from.html) now logs an error during the insert phase on the [`SQL_EXEC`](../v22.1/logging.html#sql_exec) logging channel. [#95175][#95175]
+- If `copy_from_retries_enabled` is set, [`COPY`](../v22.1/copy-from.html) is now able to retry certain safe circumstances. For example, when `copy_from_atomic_enabled` is false, there is no transaction running `COPY` and the error returned is retriable. [#95505][#95505]
```

Comment:
Is `copy_from_retries_enabled` a session setting?  If so I suggest editing to something like

> the `copy_from_retries_enabled` [session setting](https://www.cockroachlabs.com/docs/v22.1/set-vars)

Later on in the sentence I see a missing article "is able to retry _in_ certain safe circumstances"  (could also be 'under' I think)

Reply:
i think it probably is but it's not documented anywhere that i can find... same with the kv cluster setting below this. should i just link to the session/cluster settings pages anyway?

Reply:
going to merge as-is, can follow up if necessary